helm/prometheus-rules/templates/platform/atlas/recording-rules/monitoring.resource-usage-estimation.rules.yaml

```diff
@@ -9,7 +9,14 @@ spec:
   groups:
   - name: monitoring.resource-usage-estimation.recording
     rules:
+    {{- if .Values.mimir.enabled }}
+    - expr: (count({__name__=~".+"}) by (cluster_id, job) / on(cluster_id) group_left cortex_ingester_active_series{container="ingester"}) * on(cluster_id) group_left sum(container_memory_usage_bytes{container="ingester", namespace="mimir"}) by (cluster_id)
```

Comment:
Shouldn't we sum `cortex_ingester_active_series{container="ingester"}`?

Reply:
You're right, in our case we'd better sum as it'll all about the ingesters anyway. 
Updated the query.